addons/mail/static/src/models/chat_window_manager/chat_window_manager.js

```diff
@@ -141,15 +141,15 @@ function factory(dependencies) {
         }
 
         /**
-         * Shift provided chat window to the left on screen.
+         * Swap provided chat window with the previous one.
```

Comment:
Would keep "shift". "swap" is already another op on chat window, and it's confusing to use this verb here when it doesn't use swap. Also makes more sense that verb used in method name matches verb used in comment. Just rename "right/left" with "previous"/"next".

Reply:
"Shift provided chat window to the next on screen" ... it seems not correct to me ... u sure ?

Reply:
For instance:
_"Shift provided chat window to next visible index, which swap visible order of this chat window and the following visible one."_

For shift previous:
_"Shift provided chat window to previous visible index, which swap visible order of this chat window and the preceding visible one."_

Reply:
So can have "swap" in comment, but just to clarify what it means. Method starts with "shift", so it's expected the comment to also use "shift".